bug: Makefile uninstalls onefetch instead of provok

### The problem:
Running ```make uninstall``` results in an error.
```
cargo uninstall onefetch
error: package ID specification `onefetch` did not match any packages
make: *** [Makefile:8: uninstall] Error 101
```
### The Solution:
Replace the removed package with provok

--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ install:
 	cargo install --path "."
 
 uninstall:
-	cargo uninstall onefetch
+	cargo uninstall provok
 
 clean:
 	cargo clean

